exercises/G-numbers/exercise.js

```diff
@@ -1 +1,11 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+
+// Numbe of students
+const numberOfStudents = 15;
+ //Number of Mentors
+const numberOfMentors = 8;
+
+const totalOfStudentsAndMentors = numberOfStudents + numberOfStudents;
+console.log(totalOfStudentsAndMentors);
+// to practicing template litteral/ backticks 
+console.log(`There are ${numberOfStudents} students, and ${numberOfMentors} Mentors, and in total they are ${totalOfStudentsAndMentors} peaple.`)
```

Comment:
Great work! Please can you correct a spelling error -> people. ;)

Reply:
good eyes!😂. Thanks